services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java

```diff
@@ -537,7 +548,7 @@ public Response checkFileInfo(@Context UriInfo uriInfo, @Context HttpServletRequ
       verifyProofKey(request);
     } catch (ProofKeyValidationException e) {
       if (LOG.isDebugEnabled()) {
-        LOG.debug("Proof key validation failed for checkFileInfo", e);
+        LOG.debug("Proof key validation failed for checkFileInfo");
       }
       return Response.status(Status.INTERNAL_SERVER_ERROR)
```

Comment:
If it's server error, we need log it as error or warn message and details to be able answer user problem reports.

Reply:
It's client error that means the request came not from WOPI client (missing proof key headers). But in that case, WOPI requires 500 status code for the response.